### PR TITLE
Add note about using containers/common

### LINKED ISSUE #41: Consider migrating this repository into github.com/containers/common
The library `containers/common` already contains the bits for AppArmor and Capabilities so I think it would be a good idea to migrate this library into it as well. A benefit would be to unify the CI and reduce the amount of vendored dependencies from conumsers like CRI-O.

Referencing discussion in https://github.com/containers/common/issues/270.

### PR DESCRIPTION
**Pull request by saschagrunert**
I suggest making this repository read-only afterwards.

Fixes #41

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,29 +1,16 @@
...
 - IRC: #[containers](irc://irc.freenode.net:6667/#containers) on freenode.net
 
-[cri-o]:   https://github.com/kubernetes-incubator/cri-o/pulls
+[cri-o]: https://github.com/kubernetes-incubator/cri-o/pulls
 [buildah]: https://github.com/projectatomic/buildah
-[podman]:  https://github.com/projectatomic/podman
-[docker]:  https://github.com/docker/docker
-[runc]:    https://github.com/opencontainers/runc
-
+[podman]: https://github.com/projectatomic/podman
```
Comment (rhatdan) [inline README.md]: Could you change this link to containers/podman

File: README.md [hunk 2]
```diff
@@ -1,29 +1,16 @@
...
-When developing this library, please use `make` (or `make … BUILDTAGS=…`) to take advantage of the tests and validation.
+[0]: https://github.com/containers/common
 
 ## Contact
 
 - IRC: #[containers](irc://irc.freenode.net:6667/#containers) on freenode.net
 
-[cri-o]:   https://github.com/kubernetes-incubator/cri-o/pulls
+[cri-o]: https://github.com/kubernetes-incubator/cri-o/pulls
 [buildah]: https://github.com/projectatomic/buildah
```
Comment (rhatdan) [inline README.md]: containers/buildah

**Review by rhatdan — APPROVED** [review]
LGTM, with a couple of nits.

**Comment by rhatdan** [comment]
@pcmoore @vrothberg @mheon PTAL

File: README.md (hunk 1 above)
Comment (saschagrunert) [reply README.md]: Fixed :+1:

File: README.md (hunk 2 above)
Comment (saschagrunert) [reply README.md]: Fixed :+1:

**Comment by mheon** [comment]
LGTM

File: README.md [hunk 3]
```diff
@@ -1,29 +1,15 @@
...
-   make - Generates seccomp.json file, which contains the whitelisted syscalls that can be used by container runtime engines like [CRI-O][cri-o], [Buildah][buildah], [Podman][podman] and [Docker][docker], and container runtimes like OCI [Runc][runc] to controll the syscalls available to containers.
-
-### Supported build tags
-
-   `seccomp`
-   
-## Contributing
-
-When developing this library, please use `make` (or `make … BUILDTAGS=…`) to take advantage of the tests and validation.
+[0]: https://github.com/containers/common
```
Comment (pcmoore) [inline README.md]: Just put the URL directly above, no need to use a footnote/endnote.

Comment (saschagrunert) [reply README.md]: Fixed :+1:

**Comment by pcmoore** [comment]
Looks good to me.

`Acked-by: Paul Moore <paul@paul-moore.com>`

**Pull request merged**